src/containers/nav/desktop/NavColumn.module.css

```diff
@@ -307,6 +307,16 @@ svg.logo path {
   color: var(--secondary);
 }
 
+.updateDot {
```

Comment:
you'll hate me for this, but updatedot screams to me that it wants to be its own component that lives in /general.
 i know it would be a very trivial one for now, but it could do the mobile management on its own

const mobile = useIsMobile()
and set styling accordingly.

it'll not do much else for now, but maybe some day it will grow up!